Jenkinsfile

```diff
@@ -253,8 +257,14 @@ def sendKibanaMail() {
   }
 }
 
-def runbld(script) {
-  sh '#!/usr/local/bin/runbld\n' + script
+def runbld(script, noJunit = true) {
```

Comment:
Looks like this parameter is unnecessary, and just makes things a little confusing (`noX = true`, use `false` to enable...)

Reply:
It's currently used by the "legacy" runner, since that doesn't require doing the junit processing in a separate step.

You're right though, I renamed to be less confusing